heron/spi/src/java/com/twitter/heron/spi/utils/ShellUtils.java

```diff
@@ -109,14 +108,14 @@ private static int runSyncProcess(
     final StringBuilder pStdErr = stderr == null ? new StringBuilder() : stderr;
 
     // Log the command for debugging
-    LOG.log(Level.FINE, "Process command: `$ {0}`", Arrays.toString(cmdline));
+    LOG.log(Level.INFO, "Process command: ``{0}''", String.join(" ", cmdline));
     ProcessBuilder pb = getProcessBuilder(isInheritIO, cmdline, workingDirectory, envs);
 
     Process process;
     try {
       process = pb.start();
     } catch (IOException e) {
-      LOG.log(Level.SEVERE, "Failed to run Sync Process ", e);
+      LOG.log(Level.SEVERE, "Failed to run sync process ", e);
```

Comment:
what is process string? do you mean the command? we already do info-level logging of command [here](https://github.com/twitter/heron/pull/1726/files/4c46bb49aeee15225a0a16b6fec5dc14037b5ceb#diff-49b803240c7fae81aca5e36ac0559735R111).